content/developer/howtos/javascript_views.rst

```diff
@@ -0,0 +1,260 @@
+=====
+Views
```

Comment:
```suggestion
Create a new view
```

Reply:
"Create a new field **component**" is a good title because it clearly says it's not about adding a new field to the model. Is there an equivalent for views? Something like "Create a new view **handler**"? Probably not "handler", but you get my point.

In the future, I believe that howtos will be properly categorized, but for now, we should play with titles to make the distinction clear.